data/helpers/tokens.js

```diff
@@ -0,0 +1,27 @@
+const jwt = require('jsonwebtoken');
+
+const jwtSecret = `-MRUjQV"vsua!:}h?LmR},&t(zXe)a@b:z4ZpUHMd!WjKG:M@*<T(A^5#.,%'&@!`;
```

Comment:
Hide this secret in a `.env` and include the file in `.gitignore` to keep it a secret.